txtdirect.go

```diff
@@ -168,6 +178,17 @@ func fallback(w http.ResponseWriter, r *http.Request, fallback string, code int,
 	http.NotFound(w, r)
 }
 
+func customResolver(c Config) net.Resolver {
+	resolver := net.Resolver{
+		PreferGo: true,
+		Dial: func(ctx context.Context, network, address string) (net.Conn, error) {
+			d := net.Dialer{}
+			return d.DialContext(ctx, "tcp", c.Resolver)
```

Comment:
Not sure we want to choose the networking type ourselves; DNS uses UDP by default if I recall correctly, and Go will probably choose for us when calling `Dial`